### ISSUE TITLE
Fix false negatives in Guides a11y checks

### ISSUE THREAD

**Issue by beechnut** [18F/TLC-crew#369]
### A description of the work
pa11y
### Point of contact on this issue
Who can we follow-up with if we have questions?

### Reproduction steps (if necessary)
Be as specific as possible

**Billable?**

- [ ] Yes
- [ ] No
If yes, tock code:

**Skills needed**

- [ ] Any human
- [ ] Design
- [ ] Content
- [ ] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

**Timeline**
Does this need to happen in the next two weeks?

- [ ] Yes
- [ ] No

How much time do you anticipate this work taking?

### Acceptance Criteria

- [ ] What does "done" look like?

ex: As an 18F staff member who is going on parental leave, I can navigate to my
latest Tock reporting period and choose billing line #1810 by filtering by
number or "extended OOO"

**Comment by igorkorenfeld** [comment]
There definitely might be some false negative, but because empty alt text (as opposed to a missing alt attribute) is correct for decorative images I am not sure if there is something we could (or should) configure for pa11y. Instead, it may need to be a part of the manual testing and reviewing. For example on the brand/images page, I think the purpose of these images was to serve as a sort of cover image before the download button. However, I could see an argument for including alt text on these two. 

Where this will come up often is in the De-risking Guide which uses many decorative images that I believe will need blank alt text.
